src/CreatesMenuItems.php

```diff
@@ -93,11 +93,33 @@ public function &add(string $title, $config = null)
 
             return $this;
         }
+        
+
+		// Is it another Menu?
+		if(starts_with($config, 'menu:') || starts_with($config, 'sub-menu:')) {
+			$slug = $item['__slug__'] = snake_case($title);
+
+			if(starts_with($config, 'menu:')) {
+				$items =& $this->_items;
+				$menu = app('menu')->get(str_replace_first('menu:', '', $config));
+			} else {// if(starts_with($config, 'sub-menu:')) {
+				$items =& $item->_items;
```

Comment:
`$item->_items` is a protected item and it should stay that way. I'd suggest moving the functionality away from the `add` method and move it to a `merge` method under the trait `CreatesMenuItems`.